lib/src/github_sign_in_page.dart

```diff
@@ -22,53 +22,40 @@ class GitHubSignInPage extends StatefulWidget {
 }
 
 class _GitHubSignInPageState extends State<GitHubSignInPage> {
-  final FlutterWebviewPlugin _wv = FlutterWebviewPlugin();
-
   static const String _userAgentMacOSX =
       "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.88 Safari/537.36";
 
   @override
   void initState() {
     super.initState();
-
-    _wv.onUrlChanged.listen((url) {
-      if (url.startsWith(widget.redirectUrl)) {
-        Navigator.of(context)
-            .pop(url.replaceFirst("${widget.redirectUrl}?code=", "").trim());
-      } else if (url.contains("error=")) {
-        Navigator.of(context).pop(
-          Exception(Uri.parse(url).queryParameters["error"]),
-        );
-      }
-    });
   }
 
   @override
   Widget build(BuildContext context) {
-    return WebviewScaffold(
-      url: widget.url,
-      appBar: new AppBar(
-        title: Text(widget.title),
-      ),
-      userAgent: widget.userAgent ?? _userAgentMacOSX,
-      withZoom: false,
-      withLocalStorage: false,
-      withJavascript: true,
-      clearCache: widget.clearCache,
-      clearCookies: widget.clearCache,
-      initialChild: Container(
-        color: Theme.of(context).primaryColor,
-        child: const Center(
-          child: CircularProgressIndicator(),
+    return Scaffold(
+        appBar: new AppBar(
+          title: Text(widget.title),
         ),
-      ),
-    );
+        body: WebView(
+          initialUrl: widget.url,
+          userAgent: widget.userAgent ?? _userAgentMacOSX,
+          onPageFinished: (url) {
+            if (url.startsWith(widget.redirectUrl)) {
+              Navigator.of(context).pop(
+                  url.replaceFirst("${widget.redirectUrl}?code=", "").trim());
+            } else if (url.contains("error=")) {
+              Navigator.of(context).pop(
+                Exception(Uri.parse(url).queryParameters["error"]),
+              );
+            }
+          },
+        ));
   }
 
   @override
   void dispose() {
     super.dispose();
-    _wv.dispose();
-    _wv.close();
+    // _wv.dispose();
```

Comment:
@Gene-Dana I would suggest to  remove rather than comment these lines if they are not  necessary any more.